pkg/kubelet/apis/config/fuzzer/fuzzer.go

```diff
@@ -61,6 +61,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 			obj.HealthzPort = 10248
 			obj.HTTPCheckFrequency = metav1.Duration{Duration: 20 * time.Second}
 			obj.ImageMinimumGCAge = metav1.Duration{Duration: 2 * time.Minute}
+			obj.ImageMaximumGCAge = metav1.Duration{}
```

Comment:
Should you just use the default of 0s here?

Reply:
I believe that is what is done by this. Duration is just a int64:
```
type Duration int64
```